### PR TITLE
Project migration

### PR DESCRIPTION
**Pull request by allanger**
Project should be migrated from the kci org

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -7,22 +7,35 @@ on:
...
+        run: echo "LINK=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" >> $GITHUB_ENV
       - name: Build and push
         id: docker_build
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v3
         with:
           push: true
-          tags: ghcr.io/kloeckner-i/cloudish-sql:${{ github.event.release.tag_name }}
+          tags: |
+            ghcr.io/kloeckner-i/cloudish-sql:${{ github.event.release.tag_name }}
```
Comment (hyunysmile) [inline .github/workflows/release.yaml]: there is still kloeckner-i

Comment (allanger) [reply .github/workflows/release.yaml]: Updated

Comment (allanger) [reply .github/workflows/release.yaml]: @hyunysmile Could you have a look one more time? :)

Comment (allanger) [reply .github/workflows/release.yaml]: Maybe @dabde has some time to check it out? 
It's just blocking the Charts PR, and hence the migration 🙃

**Review by dabde — APPROVED** [review]

**Pull request merged**